src/_includes/api/functions.html

```diff
@@ -8306,6 +8306,24 @@
              </span>
             </span>
            </em>
+           ,
```

Comment:
Changes to this file are in Matt's PR #1087 FYI. He caught an issue where they weren't being generated at release time.